spyder/plugins/ipythonconsole/widgets/client.py

```diff
@@ -158,7 +146,6 @@ def __init__(self, parent, id_,
         # To keep a reference to the page to be displayed
         # in infowidget
         self.info_page = None
-        self._before_prompt_is_ready()
```

Comment:
How is the info page shown now if this method is not called here?

Reply:
It is called in connect_kernel